rustls/src/manual/fips.rs

```diff
@@ -1,3 +1,4 @@
+// ORPHANED DUE TO FEATURE [FEATURE REMOVED FROM THIS FORK] fips
```

Comment:
```suggestion
// [FEATURE REMOVED FROM THIS FORK] ORPHANED MANUAL MODULE
```

still ugly, hopefully more consistent now